build-config/make/sysroot.make

```diff
@@ -52,6 +55,10 @@ $(DEV_SYSROOT_INIT_STAMP): $(TREE_STAMP) | $(XTOOLS_BUILD_STAMP)
 	$(Q) echo "==== Preparing a new development sysroot ===="
 	$(Q) rm -rf $(DEV_SYSROOT)
 	$(Q) cp -a $$($(CROSSBIN)/$(CROSSPREFIX)gcc -print-sysroot) $(DEV_SYSROOT)
+	$(Q) rsync -rl $(XTOOLS_INSTALL_DIR)/$(TARGET)/$(TARGET)/lib $(DEV_SYSROOT)
+       ifeq ($(ARCH),$(filter $(ARCH),arm64 x86_64))
+		$(Q) rsync -rl $(XTOOLS_INSTALL_DIR)/$(TARGET)/$(TARGET)/lib64 $(DEV_SYSROOT)
+       endif
```

Comment:
That patch helps, along with a couple of other clean ups.

The main problem now is the none of the Accton x86 boards are able to compile busybox.  There are errors like this during the patch phase:
```
==== Patching Busybox ====
`/work/monster-15/curt/trees/onie-cn/onie/machine/accton/accton_wedge_16x/busybox/patches/syseeprom-io-interface.patch' -> `/work/monster-15/curt/trees/onie-cn/onie/build/accton_wedge_16x-r0/busybox/patch/syseeprom-io-interface.patch'
Initialized empty Git repository in /work/monster-15/curt/trees/onie-cn/onie/build/accton_wedge_16x-r0/busybox/busybox-1.25.1/.git/
Checking for changes in the working directory ... done
Importing patch "u-boot-env-tools.patch" ... done
Importing patch "feature-iorw-command.patch" ... done
Importing patch "dhcp-additional-options.patch" ... done
Importing patch "feature-flash-erase-command.patch" ... done
Importing patch "feature-onie-syseeprom-command.patch" ... done
Importing patch "feature-onie-syseeprom-disk.patch" ... done
Importing patch "feature-onie-syseeprom-sysfs.patch" ... done
Importing patch "syseeprom-io-interface.patch" ... error: patch failed: miscutils/Kbuild.src:51
error: miscutils/Kbuild.src: patch does not apply
stg import: Diff does not apply cleanly
make: *** [/work/monster-15/curt/trees/onie-cn/onie/build/accton_wedge_16x-r0/stamp/busybox-patch] Error 2
```

Seeing problems like that for accton_wedge_16x-r0, accton_as7712_32x, accton_as5712_54x, accton_as6712_32x, accton_as5812_54t.  @pankajbansal3073  - as a separate commit please make sure these boards can compile.